Report_templates/qwtimeseries.Rmd

```diff
@@ -46,21 +46,23 @@ totalP_title <- "TOTAL PHOSPHORUS CONCENTRATION\nIN MILLIGRAMS PER LITER"
 chlorophyll_title <- "CHLOROPHYLL CONCENTRATION\nIN MICROGRAMS PER LITER"
 secchi_title <- "SECCHI DEPTH, IN METERS"
 trophic_title <- "TROPHIC STATE INDEX"
+top_margin <- 0
+bottom_margin <- 2
```

Comment:
no need to add these here - they are the accepted defaults inside the functions.